# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtkmm12/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ur gtkmm-1.2.10/src/clist.gen_h gtkmm-1.2.10-gcc4/src/clist.gen_h
--- gtkmm-1.2.10/src/clist.gen_h	2001-08-16 19:56:49.000000000 +0200
+++ gtkmm-1.2.10-gcc4/src/clist.gen_h	2005-09-23 12:10:31.888869901 +0200
@@ -60,7 +60,7 @@
 class Cell 
   {
     protected:
-      friend class CList;
+      friend class Gtk::CList;
       friend class Row;
 
       GtkCList    *parent_;
diff -ur gtkmm-1.2.10/src/radiobutton.gen_h gtkmm-1.2.10-gcc4/src/radiobutton.gen_h
--- gtkmm-1.2.10/src/radiobutton.gen_h	2001-08-04 19:36:51.000000000 +0200
+++ gtkmm-1.2.10-gcc4/src/radiobutton.gen_h	2005-09-23 12:10:48.004135574 +0200
@@ -36,7 +36,7 @@
   //- It should not be called with new.
   class Group
     {
-      friend class RadioButton;
+      friend class Gtk::RadioButton;
       GSList *group_;
       void* operator new(size_t);
       void add(RadioButton& item);
diff -ur gtkmm-1.2.10/src/radiomenuitem.gen_h gtkmm-1.2.10-gcc4/src/radiomenuitem.gen_h
--- gtkmm-1.2.10/src/radiomenuitem.gen_h	2001-08-04 19:36:51.000000000 +0200
+++ gtkmm-1.2.10-gcc4/src/radiomenuitem.gen_h	2005-09-23 12:11:04.526332198 +0200
@@ -34,7 +34,7 @@
   //- It should not be called with new.
   class Group 
     {
-      friend class RadioMenuItem;
+      friend class Gtk::RadioMenuItem;
       GSList *group_;
       void* operator new(size_t);
       void add(RadioMenuItem& item);
